jax/experimental/jax2tf/saved_model_lib.py

```diff
@@ -21,47 +21,32 @@
 
 from typing import Callable, Sequence, Optional
 
-from jax.experimental import jax2tf # type: ignore[import]
+from jax.experimental import jax2tf  # type: ignore[import]
 import tensorflow as tf  # type: ignore[import]
 
 
-def save_model(jax_fn: Callable,
-               params,
-               model_dir: str,
-               *,
-               input_signatures: Sequence[tf.TensorSpec],
-               shape_polymorphic_input_spec: Optional[str] = None,
-               with_gradient: bool = False,
-               enable_xla: bool = True,
-               compile_model: bool = True):
-  """Saves the SavedModel for a function.
+def convert_and_save_model(jax_fn: Callable,
+                           params,
+                           model_dir: str,
+                           *,
+                           input_signatures: Sequence[tf.TensorSpec],
```

Comment:
Bit confused by this, each signature should be a Pair and the elements of the pair should be optionally nested tf.TensorSpec no?

    Params = Inputs = Nest[tf.TensorSpec]
    PredictSignature = Tuple[Params, Inputs]
    PredictSignatures = Sequence[PredictSignature]

Reply:
The input signatures are just for the input, not including the parameters. 

I wish I can use `Nest[tf.TensorSpec]` notation, but I don't know of an easy way to define the `Nest` generic hint, so I just use the type hint as if we have a leaf nest.